.circleci/config.yml

```diff
@@ -140,7 +140,7 @@ workflows:
               nightly-test: [true]
               ios: [*latest-ios]
           requires:
-            - test << matrix.lib >> iOS 13.7
+            - test << matrix.lib >> iOS 14.0
       - run-tests:
```

Comment:
There is no beta Xcode available at the moment so we can ignore/comment out/delete this.